examples/catheter/conf/catheter.yaml

```
@@ -0,0 +1,93 @@
+defaults:
+  - ppsci_default
+  - TRAIN: train_default
+  - TRAIN/ema: ema_default
+  - TRAIN/swa: swa_default
+  - EVAL: eval_default
+  - hydra/job/config/override_dirname/exclude_keys: exclude_keys_default
+  - _self_
+
+hydra:
+  run:
+    # dynamic output directory according to running time and override name
+    dir: outputs_geofno/${now:%Y-%m-%d}/${now:%H-%M-%S}/${hydra.job.override_dirname}
+  job:
+    name: ${mode} # name of logfile
+    chdir: false # keep current working directory unchanged
+  callbacks:
+    init_callback:
+      _target_: ppsci.utils.callbacks.InitCallback
+  sweep:
+    # output directory for multirun
+    dir: ${hydra.run.dir}
+    subdir: ./
+
+# general settings
+mode: train # running mode: train/eval
+seed: 42
+output_dir: ${hydra:run.dir}
+log_freq: 100
+data_path: ./data
+
+TRAIN_DATA:
+  x_path: "${data_path}/training/x_1d_structured_mesh.npy"
+  y_path: "${data_path}/training/y_1d_structured_mesh.npy"
+  para_path: "${data_path}/training/data_info.npy"
+  output_path: "${data_path}/training/density_1d_data.npy"
+  n_data: 3000
+  n: 1000
+  s: 2001
+
+TEST_DATA:
+  x_path: "${data_path}/test/x_1d_structured_mesh.npy"
+  y_path: "${data_path}/test/y_1d_structured_mesh.npy"
+  para_path: "${data_path}/test/data_info.npy"
+  output_path: "${data_path}/test/density_1d_data.npy"
+  n_data: 300
+  n: 100
+  s: 2001
+
+# model settings
+MODEL:
+  modes: 64
+  width: 64
+  padding: 100
+  input_channel: 2
+  output_np: 2001
+
+# training settings
+TRAIN:
+  use_pretrained_model: False
+  pretrained_model_path: "https://dataset.bj.bcebos.com/PaddleScience/2024%20AI-aided%20geometric%20design%20of%20anti-infection%20catheters/init_GeoFno.pdparams"
```

模型训练的参数应该来自随机初始化，而不是预训练模型，否则换一个场景或者微调网络结构可能导致报错

```suggestion
  pretrained_model_path: null
```

---

好的好的，已经按照要求修改了，请review